### PR TITLE
trainee_finish_subject

### PR DESCRIPTION
**Pull request by hienbx94**
I have created a pull request. Please check it.

### CODE REVIEW

**Comment by lannp** [comment]
OK

File: app/controllers/subjects_controller.rb [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  before_action :check_status, only: :update
+
+  def update
+    if @user_subject.update_attributes status: true
+      flash[:success] = flash_message "updated"
+    else
+      flash[:danger] = flash_message "not_updated"
+    end
+    respond_to do |format|
+      format.html { redirect_to [@user_course, @user_subject] }
```
Comment (hellovietnam93) [inline app/controllers/subjects_controller.rb]: remove space

File: app/views/subjects/update.js.erb [hunk 2]
```diff
@@ -0,0 +1 @@
+$("#finish_form").html("<%= escape_javascript(render('user_subjects/finished')) %>");
```
Comment (hellovietnam93) [inline app/views/subjects/update.js.erb]: using " instead of '

File: app/views/user_subjects/_finish.html.erb [hunk 3]
```diff
@@ -0,0 +1,3 @@
+<%= link_to t("subjects.finish_subject"), user_course_user_subject_finish_subject_path(
+  @user_course, @user_subject, Settings.finish), remote: true, method: :patch, class: "btn btn-primary" %>
+
```
Comment (hellovietnam93) [inline app/views/user_subjects/_finish.html.erb]: convention

File: app/views/user_subjects/_finish.html.erb [hunk 4]
```diff
@@ -0,0 +1,3 @@
+<%= link_to t("subjects.finish_subject"), user_course_user_subject_finish_subject_path(
+  @user_course, @user_subject, Settings.finish), remote: true, method: :patch, class: "btn btn-primary" %>
```
Comment (hellovietnam93) [inline app/views/user_subjects/_finish.html.erb]: 2 lines

**Comment by hellovietnam93** [comment]
OK

**Comment by vietanhbkaaa** [comment]
OK

**Pull request merged**